### PR TITLE
Update helpMarkDown for AndroidSigningV3 apksignerArguments

### PR DESCRIPTION
**Pull request by anatolybolshakov**
**Task name**: AndroidSigningV3

**Description**: reworked https://github.com/microsoft/build-task-team/issues/212

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AndroidSigningV3/task.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 175,
+        "Minor": 176,
```
Comment (alexander-smolyakov) [inline Tasks/AndroidSigningV3/task.json:15]: Could you please bump up a minor version in `task.lock.json` as well?

Comment (anatolybolshakov) [reply Tasks/AndroidSigningV3/task.json:15]: Thanks! Updated it also

Comment (anatolybolshakov) [reply Tasks/AndroidSigningV3/task.json:15]: Thanks! Updated it also

**Review by alexander-smolyakov — APPROVED** [review]
LGTM, thanks!

**Review by alexander-prozorov — APPROVED** [review]
LGTM, thanks!

**Pull request merged**